### PR TITLE
Bug 1206309: Display tweaks for history drop downs.

### PR DESCRIPTION
**Pull request by stephaniehobson**
{{embedCompatTable('web-css-display')}} will give you a table with history drop downs.

Corresponding change to kumascript macro: https://gist.github.com/stephaniehobson/0d0955e02533e040ba7c

### CODE REVIEW

File: kuma/static/js/wiki-compat-tables.js [hunk 1]
```diff
@@ -19,6 +19,8 @@
     // Private var to assign IDs to history for accessibility purposes
     var historyCount = 0;
 
+    // The open button template
+    var $historyLink = $('<button title="Open" class="bc-history-link only-icon" tabindex="-1"><span>Open</span><i class="ic-history" aria-hidden="true"></i></button>');
```
Comment (darkwing) [inline kuma/static/js/wiki-compat-tables.js]: May want to use `gettext` for "Open"

File: kuma/static/stylus/components/compat-tables/bc-history-link.styl [hunk 2]
```diff
@@ -16,11 +16,12 @@ History-link
     font-size: 12px;
     border-top: 1px solid rgba($bc-color-border, 0.3);
     line-height: 1;
-
-    /* needs additional specificity to over ride rule giving it an underline */
-    a& {
-        text-decoration: none;
-    }
+    // over ride MDN defaults
```
Comment (darkwing) [inline kuma/static/stylus/components/compat-tables/bc-history-link.styl]: "override" is one word  (/me hides)

File: kuma/static/js/wiki-compat-tables.js [hunk 3]
```diff
@@ -53,8 +55,10 @@
                     'aria-expanded': false,
                     'aria-controls': historyId
                 });
-                $td.find('.bc-history').attr('id', historyId);
-                $td.find('.bc-history-link').attr('tabIndex', -1);
+                var $history = $td.find('.bc-history');
```
Comment (darkwing) [inline kuma/static/js/wiki-compat-tables.js]: You may want to move this new line to right after `var $td = $(this);` since JavaScript vars are hoisted up.

**Comment by stephaniehobson** [comment]
Updated.

**Comment by darkwing** [comment]
R+

**Pull request merged**